pages/index.tsx

```diff
@@ -1,16 +1,38 @@
 import Head from 'next/head';
+import Link from 'next/link';
+import { useUser } from '@auth0/nextjs-auth0/client';
+import { useState } from 'react';
+
+export default function Home(this: any) {
+  const { user, error, isLoading } = useUser();
 
-export default function Home() {
   return (
     <>
+      {console.log(user?.email)}
```

Comment:
Feel free to remove this since we will want to exclude console logs in production. For future reference, this console log being inside the JSX here can be put in the component function body too, and might be a touch easier to work with.